### DISCUSSION TITLE
How to implement mysql in opentcs?

### DISCUSSION THREAD

**Discussion by xfyecn** [openTCS/opentcs#18] [General]
Hi Develop,

      Here are two questions about opentcs?
      1)How to describe the map information and routing information in opentcs?
      2)How to store agv's map information and routing information in MySQL, i.e., how to implement MySQL in opentcs?
      Cloud you give me some clues or example?  
      Thanks in advance,
      Sincerely, 
      Xianfeng Ye

**Comment by swltr** [comment]
I think you need to provide some more information about your use case. What are you trying to achieve? What do you need to describe map and routing information for? What are you integrating MySQL for? Without this information about the problem you are trying to solve, it is difficult to tell you what a good solution might be.

**Comment by freekevinCode** [comment]
Chinese？ 可以把plant model 转换为json  存储到mysql指定表的json类型字段中
